lib/get-outputs.js

```diff
@@ -47,8 +47,13 @@ module.exports = function(dir, options) {
               if (result && result.output) {
                 const parsed = JSON.parse(result.output);
 
-                return Object.keys(parsed).reduce((memo, key) =>
-                  Object.assign(memo, { [key]: parsed[key].value }), {});
+                const addValue = (key, value, acc) => {
+                  return typeof value === 'object' && options.flatten
```

Comment:
`typeof null` is `object`, do you think anything coming back might blow up?

Reply:
Hmm, good point. TF does not return top-level outputs that are nulls or empty strings, but nested values can be nulls. I updated the check and test.